README.md

```diff
@@ -55,6 +55,20 @@ when you run into errors with a CloudFormation stack:
 
 ## Installation
 
+### MacOS: Binary Install via Homebrew
+
+Use Unbounce's custom Homebrew Tap to install iidy.
```

Comment:
I've open-sourced this already so I'll add these instructions on Crystal internally.

Reply:
Yes, this is intentional.  Homebrew taps are public repositories.

https://github.com/unbounce/homebrew-taps is a public repository.  There's nothing wrong with these instructions being read by the public.